logger/request.go

```diff
@@ -124,6 +129,22 @@ func (h Headers) MarshalLogArray(enc zapcore.ArrayEncoder) error {
 	return nil
 }
 
+func maskAuthorizationHeader(value string) string {
+	maskChar := "***"
+	maskData := MaskData{
+		FirstCharsToShow: 2,
+		LastCharsToShow:  3,
+		KeepSameLength:   false,
+		prefixesToSkip:   []string{"pk_test_", "pk_", "sk_test_", "sk_"},
```

Comment:
@yanyi-wego  I think we should sort this array by length from longest to shortest

Reply:
Done in f1d636c.